src/fermilib/utils/_operator_utils_test.py

```diff
@@ -14,12 +14,15 @@
 from __future__ import absolute_import
 
 import numpy
+import os
 import unittest
 
+from fermilib.config import *
 from fermilib.ops import *
 from fermilib.transforms import jordan_wigner, get_interaction_operator
 from fermilib.utils import (eigenspectrum, commutator,
-                            count_qubits, is_identity)
+                            count_qubits, is_identity,
+                            save_operator, load_operator, get_file_path)
 
 from projectq.ops import QubitOperator
```

Comment:
Can you possibly add another test? Maybe one that tests the save when file name is "None". We are currently trying to only accept PRs that raise the test coverage or keep it the same and this one lowers test coverage by 0.06%.